test/Microsoft.Extensions.Logging.Test/FormattedLogValuesTest.cs

```diff
@@ -17,6 +17,7 @@ public class FormattedLogValuesTest
         [InlineData("arg1 arg2", "{Start} {End}", new object[] { "arg1", "arg2" })]
         [InlineData("arg1     arg2", "{Start,-6} {End,6}", new object[] { "arg1", "arg2" })]
         [InlineData("0064", "{Hex:X4}", new object[] { 100 })]
+        [InlineData("8,765", "{Number:#,#}", new object[] {8765.4321})]
```

Comment:
Please add `[InlineData(" 8,765", "{Number,6:#,#}", new object[] { 8765.4321 })]` test case and I'll merge it.